src/test/config.sol

```diff
@@ -1849,5 +1849,30 @@ contract Config {
             calc_cut:     99_00,
             offboarding:  false
         });
+        afterSpell.collaterals["ALLOCATOR-SPARK-A"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      10 * MILLION,
+            aL_gap:       2_500_000,
+            aL_ttl:       24 hours,
+            line:         0,
+            dust:         0,
+            pct:          520,
+            mat:          10000,
+            liqType:      "",
+            liqOn:        false,
+            chop:         0,
+            dog_hole:     0,
+            clip_buf:     0,
+            clip_tail:    0,
+            clip_cusp:    0,
+            clip_chip:    0,
+            clip_tip:     0,
+            clipper_mom:  0,
+            cm_tolerance: 0,
+            calc_tau:     0,
+            calc_step:    0,
+            calc_cut:     0,
+            offboarding:  false
+        });
```

Comment:
Are the base test for regular ilks enough to validate allocators?
Maybe it would be better to create a specialized version of it, calling `afterSpell.allocators` as well.